.github/workflows/process_test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"        
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: "main"
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
There are some Github docs on encrypted secrets [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets). 

Right now, I create repo-only secrets rather than organization secrets. I like using them from the perspective that, once created, their contents cannot be accessed through Github by any user (even me as the creator!)

The Github token referred to here is just an API token, which we can call in workflows with the allocated name `GITHUB_TOKEN`.